sycl/include/CL/sycl/ONEAPI/atomic_ref.hpp

```diff
@@ -453,13 +453,17 @@ class atomic_ref_impl<
 
   T fetch_add(T operand, memory_order order = default_read_modify_write_order,
               memory_scope scope = default_scope) const noexcept {
+#if defined(__SYCL_DEVICE_ONLY__) && !defined(__NVPTX__)
+    return detail::spirv::AtomicFAdd(ptr, scope, order, operand);
```

Comment:
Can we implement `fetch_sub()` as something like
```
auto Neg = detail::spirv::FNegate(operand); // not sure we have a thing like this.
return detail::spirv::AtomicFAdd(ptr, scope, order, Neg);
```
?

Reply:
```
return detail::spirv::AtomicFAdd(ptr, scope, order, -operand);
```

Reply:
Done, at least as a draft.